From 1fd55647633dbafc3e49184c3f5ca45b541a116a Mon Sep 17 00:00:00 2001
From: Ashish Kumar <Ashish.Kumar@nxp.com>
Date: Mon, 19 Feb 2018 14:14:52 +0530
Subject: [PATCH 002/107] armv8: ls1088a: Add clock related function in CONFIG_SPL_BUILD

get_board_ddr_clk(), get_board_sys_clk() and if_board_diff_clk() is
now available for SPL build.

Signed-off-by: Ashish Kumar <Ashish.Kumar@nxp.com>
Reviewed-by: York Sun <york.sun@nxp.com>
---
 board/freescale/ls1088a/ls1088a.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/board/freescale/ls1088a/ls1088a.c b/board/freescale/ls1088a/ls1088a.c
index 56e454f..161ea6d 100644
--- a/board/freescale/ls1088a/ls1088a.c
+++ b/board/freescale/ls1088a/ls1088a.c
@@ -168,6 +168,7 @@ int checkboard(void)
 
 	return 0;
 }
+#endif
 
 bool if_board_diff_clk(void)
 {
@@ -221,7 +222,6 @@ unsigned long get_board_ddr_clk(void)
 
 	return 66666666;
 }
-#endif
 
 int select_i2c_ch_pca9547(u8 ch)
 {
-- 
1.7.1

